meetings/2023-02-23.md

```diff
@@ -0,0 +1,132 @@
+---
+parent: Meetings
+title: "2023-02-23"
+---
+
+# AWSF TAC Meeting - February 23, 2023
+
+## Voting member attendance
+
+### Premier member representatives
+- [  ] Bill Roberts - Adobe Inc.
+- [  ] Brian Cipriano - Google LLC
+- [  ] Cory Omand - The Walt Disney Studios
+- [  ] Eric Enderton - NVIDIA Corporation
+- [  ] Eric Reinecke - Netflix, Inc.
+- [  ] Gordon Bradley - Autodesk
+- [  ] Greg Denton - Microsoft Corporation
+- [  ] Kimball Thurston - Weta Digital Ltd
+- [  ] Larry Gritz - Sony Pictures Imageworks
+- [  ] Mark Visser - Unity Technologies
+- [  ] Matthew Low - DreamWorks Animation
+- [  ] Michael B. Johnson - Apple Inc.
+- [  ] Sean Looper - Amazon Web Services, Inc.
+- [  ] Sean McDuffee - Intel Corporation
+- [  ] Sean O'Connell - Advanced Micro Devices (AMD)
+- [  ] Tony Micilotta - DNEG
+
+### Project Representatives
```

Comment:
Should this section list the project affiliation rather than the company, or perhaps both?